docs/explanation/package-model.rst

```diff
@@ -1,9 +1,196 @@
-.. _package-model:
-
-=============
 Package Model
 =============
 
-.. caution::
+Because :term:`Ubuntu` is based on the community-driven :term:`Debian` project,
+:term:`Ubuntu` uses the :term:`Debian` packaging model/format.
+
+There are :ref:`SourcePackages` and :ref:`BinaryPackages`.
+
+.. _SourcePackages:
+
+Source Packages
+---------------
+
+A *source package* contains the :term:`Source` material used to build one or
+more :ref:`BinaryPackages`.
+
+A *source package* is composed of
+
+- a :file:`.dsc` (*Debian Source Control*) file,
+- one or more compressed tar files, and 
+- optionally additional files depending on the type and format of the *source package*.
+
+The *Source Control* file contains metadata about the *source package*, for instance, 
+a list of additional files, name and version, list of the binary packages it produces, 
+dependencies, a digital signature and many more fields.
+
+.. note::
+  The article :doc:`/reference/debian-dir-overview` showcases the layout of 
+  an unpacked *source package*.
+
+Source Package Formats
+~~~~~~~~~~~~~~~~~~~~~~
+
+There exist multiple formats for how the :term:`Source` is packaged. The format of a 
+*source package* is declared in the :file:`debian/source/format` file. This file should
+always exist. If this file can not be found, the :ref:`format 1.0 <SourcePackageFormat_1.0>`
+is assumed for backwards compatibility, but :manpage:`lintian(1)` will warn you about it.
+
+.. tip::
+
+    If you don't know what source format to use, you should probably pick either 
+    :ref:`3.0 (quilt) <SourcePackageFormat_3.0quilt>` or
+    :ref:`3.0 (native) <SourcePackageFormat_3.0native>`.
+
+
+.. _NativeSourcePackages:
+
+(Non-)Native Source Packages
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In most cases, a software project is packaged by external contributors
+(:term:`Maintainers <Maintainer>`) only tangentially related to the software project. 
```

Comment:
The maintainer/packager of a software packages are often 3rd parties from the perspective of a software project.

A good example is the Ubuntu tor browser. I don't know its current state, but the tor browser was once removed from the archive, because the tor project requested it. They had no control over the package version in the Ubuntu archive and were annoyed by users reporting bugs that were alre3ady fixed, because the users assumed that the package was reasonably up-to-date.

Reply:
I changed the wording with d95ce7b43232c1ae4f27b55c427d3369f63465e8